lib/perl/Genome/VariantReporting/Suite/Vep/Run.pm

```diff
@@ -65,3 +70,27 @@ sub name {
 sub result_class {
     'Genome::VariantReporting::Suite::Vep::RunResult';
 }
+
+sub __planned_errors__ {
+    my ($self, $params) = @_;
+    my $version = $params->{joinx_version};
+    my $allow_same_file = $params->{allow_same_file};
+    return $self->SUPER::__planned_errors__($params), $self->_get_joinx_version_error($version, $allow_same_file);
+}
+
+sub _get_joinx_version_error {
+    my ($self, $version, $allow_same_file) = @_;
+
+    my @errors;
+    if ($allow_same_file && (version->parse("v".$version) < version->parse("v1.11"))) {
+        push @errors, UR::Object::Tag->create(
+            type => 'error',
+            properties => ['joinx_version'],
+            desc => "Provided parameter joinx_version: $version not supporting allow-same--file option, use 1.11 and above",
```

Comment:
There's an extra `-` between `same` and `file`.  (Also, `does not support the` might sound better than `not supporting` here).